test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageControllerSpec.scala

```diff
@@ -1739,8 +1739,24 @@ class SingleDisposalsTriageControllerSpec
         }
 
         val result = performAction()
-        status(result)          shouldBe OK
-        contentAsString(result) shouldBe "Indirect disposals not handled yet"
+        checkIsRedirect(result, "/capital-gains-tax-uk-property/exit/indirect-mixed-use-cannot-use-service")
```

Comment:
to make things more future proof we tend to use the reverse routes to get urls instead of using hard coded strings, e.g. here you could use `routes.SingleDisposalTriageController.indirectBlahBlahBlah()` or whatever the method's called